compiler/src/test/java/com/squareup/anvil/compiler/codegen/ContributesBindingGeneratorTest.kt

```diff
@@ -496,30 +534,30 @@ class ContributesBindingGeneratorTest(
       interface ParentInterface2
 
       @ContributesBinding(Any::class, boundType = ParentInterface1::class) // Default case is NORMAL
-      @ContributesBinding(Any::class, boundType = ParentInterface2::class, priority = Priority.NORMAL)
-      @ContributesBinding(Unit::class, boundType = ParentInterface1::class, priority = Priority.HIGH)
-      @ContributesBinding(Unit::class, boundType = ParentInterface2::class, priority = Priority.HIGHEST)
+      @ContributesBinding(Any::class, boundType = ParentInterface2::class, priorityDeprecated = Priority.NORMAL)
+      @ContributesBinding(Unit::class, boundType = ParentInterface1::class, priorityDeprecated = Priority.HIGH)
+      @ContributesBinding(Unit::class, boundType = ParentInterface2::class, priorityDeprecated = Priority.HIGHEST)
       class ContributingInterface : ParentInterface1, ParentInterface2
       """,
       mode = mode,
```

Comment:
```suggestion
      mode = mode,
      allWarningsAsErrors = false
```

Our default value for KCT's `-Werror` is tied to the `CI` environment variable.  So, that test's compilation is failing in CI with:

```
e: warnings found and -Werror specified
w: file:[...]/Source0.kt:4:58 'Priority' is deprecated. Use the new int-based priority
w: file:[...]/Source0.kt:10:91 'Priority' is deprecated. Use the new int-based priority
w: file:[...]/Source0.kt:11:92 'Priority' is deprecated. Use the new int-based priority
w: file:[...]/Source0.kt:12:92 'Priority' is deprecated. Use the new int-based priority
```

In hindsight, that `allWarningsAsErrors` probably shouldn't be toggled automagically, and we should probably have something in our test utils that asserts `exitCode == OK` by default.  I can follow up with those changes.